modules/apps/foundation/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/site/facet/portlet/ScopeFacetBuilder.java

```diff
@@ -87,6 +91,6 @@ protected FacetConfiguration buildFacetConfiguration(Facet facet) {
 	private int _maxTerms;
 	private final ScopeFacetFactory _scopeFacetFactory;
 	private SearchContext _searchContext;
-	private String[] _selectedSites;
+	private static String[] _selectedSites;
```

Comment:
Ran into an issue where I could not reference a non-static variable from a static context on the MVCActionCommand java file.

There may be a better way to access data from this variable that I was not able to find.